test/e2e/network/network_policy.go

```diff
@@ -753,6 +989,27 @@ func createNetworkClientPod(f *framework.Framework, namespace *v1.Namespace, pod
 	return pod
 }
 
+// Patch client pod with a map value
+func updateNetworkClientPodLabel(f *framework.Framework, namespace *v1.Namespace, podName string, patchOperation string, patchPath string, patchValue map[string]string) *v1.Pod {
+	type patchMapValue struct {
+		Op    string            `json:"op"`
+		Path  string            `json:"path"`
+		Value map[string]string `json:"value"`
```

Comment:
Value can be omitted for some operations. consider having `json:"value,omitempty"` instead